ravedude/src/board.rs

```diff
@@ -1,10 +1,11 @@
 use crate::avrdude;
+use anyhow::Result;
```

Comment:
Please drop this import and refer to the type as `anyhow::Result<>` explicitly below.  This makes it easier for readers to understand where this result type is coming from.